platform/android/java/lib/src/org/godotengine/godot/Godot.java

```diff
@@ -660,10 +660,14 @@ public void onPause() {
 		}
 	}
 
+	public boolean hasClipboard() {
```

Comment:
You can make this method `private` since it's only being used by the native code.

Reply:
The existing `{get,set}Clipboard` methods are also `public`. Should I change them too?

Reply:
Good point, I overlooked them. In that case leave the scope to `public`. 

While they are not meant to be public, there could be external code already relying on these methods which would be affected if we change the scope.

Feel free to disregard the similar comment in the `3.x` PR.